utils/spacewalk-common-channels.ini

```diff
@@ -2164,3 +2164,41 @@ repo_type = deb
 checksum = sha256
 base_channels = astralinux-orel-pool-amd64
 repo_url = https://download.astralinux.ru/astra/stable/orel/repository/dists/orel/non-free/binary-amd64/
+
+[amazonlinux2-core]
+archs    =  x86_64, aarch64
+name     = Amazon Linux 2 Core %(arch)s
+label    = amazonlinux2-core-%(arch)s
+checksum  = sha256
+gpgkey_url =
+gpgkey_id = C87F5B1A
+gpgkey_fingerprint = 99E6 17FE 5DB5 27C0 D8BD  5F8E 11CF 1F95 C87F 5B1A
+repo_url = http://amazonlinux.default.amazonaws.com/2/core/latest/%(arch)s/mirror.list
+
+[amazonlinux2-extra-docker]
+archs    =  x86_64, aarch64
+name     = Amazon Extras 2 repo for Docker %(arch)s
+label    = amazonlinux2-extras-docker-%(arch)s
+base_channels = amazonlinux2-core-%(arch)s
+checksum = sha256
+gpgkey_id = C87F5B1A
+gpgkey_fingerprint = 99E6 17FE 5DB5 27C0 D8BD  5F8E 11CF 1F95 C87F 5B1A
+repo_url = http://amazonlinux.default.amazonaws.com/2/extras/docker/latest/%(arch)s/mirror.list
+
+[amazonlinux2-uyuni-client]
+name     = Uyuni Client Tools for %(base_channel_name)s
+archs    =  x86_64, aarch64
+base_channels = amazonlinux2-core-%(arch)s
+gpgkey_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/CentOS7-Uyuni-Client-Tools/CentOS_7/repodata/repomd.xml.key
+gpgkey_id = %(_uyuni_gpgkey_id)s
+gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
+repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable:/CentOS7-Uyuni-Client-Tools/CentOS_7/
```

Comment:
We really need to decide what to do here. Personally I'd add Amazon Linux to OBS and build separate client tools, as Amazon Linux is not a *clone* of CentOS7, even if it's very similar.

Besides, if we add Amazon Linux to OBS, we'll be able to support `aarch64` as well, which is being used more and more because the EC2 instances with Amazon Linux `aarch64` are cheaper.

Reply:
We can also build aarch64 using CentOS 7